src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -336,3 +336,21 @@ def _status(self):
                                            avail,
                                            " ({0})".format(why) if not avail else ""
                                        ))
+
+    @CLIWriteCommand('orchestrator host add',
```

Comment:
You don't need to add them anymore.

Reply:
Maybe is good to keep this alive ( and rm host) while the addition/removal of hosts will be implemented in each of the  orchestrator functional operations.

Reply:
Sorry for the confusion! If you rebase to master, you'll notice that these methods are already there. (lines 49 and 58)

Reply:
this comment is not addressed.

Reply:
Removed duplicated code